src/app/MessageDef/InvokeRequestMessage.h

```diff
@@ -33,9 +33,10 @@ namespace app {
 namespace InvokeRequestMessage {
 enum class Tag : uint8_t
 {
-    kSuppressResponse = 0,
-    kTimedRequest     = 1,
-    kInvokeRequests   = 2,
+    kInteractionId    = 0,
```

Comment:
interesting, spec add InteractionID recently. @bzbarsky-apple @mrjerryjohns how can we use InteractionID for Invoke?

Reply:
The idea is that you pass it to the application and it does whatever it wants with it.